CHANGELOG/CHANGELOG-4.4.md

```diff
@@ -0,0 +1,231 @@
+# Release notes for v4.4.0
+
+[Documentation](https://kubernetes-csi.github.io)
+
+# Changelog since v4.3.0
+
+## Changes by Kind
+
+### Feature
+
+- Add --max-entries flag to limit ListVolumes request entries per page (#462, @kayrus)
+
+## Dependencies
```

Comment:
Xing suggested we add a release note for k8s dependencies updates. I updated https://github.com/kubernetes-csi/external-attacher/pull/475

Reply:
Regenerated, thanks for the PR update.